### ISSUE TITLE
[QUESTION] Does everyone who clones my repository need to initialize git flow on their machine?

### ISSUE THREAD

**Issue by ThallyssonKlein** [petervanderdoes/gitflow-avh#456]
Does everyone who clones my repository need to initialize git flow on their machine?

I would like to know if it is possible to save this configuration so that it is unnecessary to do it again.

**Comment by cizordj** [comment]
That is a very good question, I am commenting to see if anyone knows that

**Comment by petervanderdoes** [comment]
Yes they have too, and no it's not possible to save the config. The configuration is stored the the local git configuration, you add a small script in the repo that would set the configuration. The same problem occurs for other git settings like `core.autocrlf`

**Comment by GustavoCinque** [comment]
No, they won't have to. Git flow is just a tool to create and manage branches using git. Every command on gitflow has a command (or more than one, in some cases) using "native" git. So no. For example, when you use "git flow feature start branch" someone not using gitflow can start a feature branch using: "git checkout develop && git checkout -b feature/branch".

**Comment by Shea690901** [comment]
I fact, considering the way github (or gitlab, or codeberg, or ...) works, with pull requests and merges, the only one who needs to use gitflow for easier use is the repo owner himself. All other just need to keep their forks up to date and use feature branches for their development before making a pull request on github against the default branch, wich should be the development branch...
